contracts/interchain-token-service/src/msg.rs

```diff
@@ -33,6 +34,13 @@ pub enum ExecuteMsg {
     /// The admin is allowed to remove the ITS address of a chain for emergencies.
     #[permission(Elevated)]
     DeregisterItsContract { chain: ChainNameRaw },
+
+    #[permission(Governance)]
+    SetChainConfig {
+        chain: ChainNameRaw,
+        max_uint: Uint256,
```

Comment:
```suggestion
        max_uint: u8,
```
Hmm, do we want to support non power-of-two aligned max values? I suppose it's more future proof even if we don't need it atm, and saves some computation

Reply:
I prefer the current approach instead of saving the powers. It also doesn't save any computation.